schema/deploy/functions/update-modified-timestamp.sql

```diff
@@ -0,0 +1,18 @@
+-- Deploy seattleflu/schema:functions/update-modified-timestamp to pg
+
+begin;
+
+create or replace function warehouse.update_modified_timestamp() returns trigger as $$
+
+    begin
+        NEW.modified = now();
+        return new;
+    end;
+$$ language plpgsql
+    security definer
+    set search_path = pg_catalog, public, pg_temp;
```

Comment:
`search_path` is also unnecessary for this function, but there's also no harm in leaving it.

Reply:
When is adding `search_path` necessary?

Reply:
It forces the `search_path` to a known value so that unqualified function/table names aren't found in an unexpected schema (like a schema controlled by the unprivileged database user). This is important because a `security definer` function will call other functions with the same privileged user it runs as. Details here: https://www.postgresql.org/docs/10/sql-createfunction.html#SQL-CREATEFUNCTION-SECURITY

Reply:
(Also happy to more concretely sketch out why this matters in person.)

Reply:
I think I get it. So because I declare the function as being part of `warehouse`, setting the `search_path` is redundant?

Reply:
Mmm, not quite. I'll explain in person tomorrow?

Reply:
That would be great!